### PR TITLE
OCLOMRS-413: Modify concept preview modal to show synonyms of concept name

### PR DESCRIPTION
**Pull request by AGMETEOR**
# JIRA TICKET NAME:
[Modify concept preview modal to show synonyms of concept name](https://issues.openmrs.org/browse/OCLOMRS-413)

# Summary:
A synonym is any valid, alternative name for the concept. This includes acronyms, abbreviations, and other names that reference the primary name. The preview concept card should show the concept's synonym as well.

### CODE REVIEW

File: src/tests/bulkConcepts/components/PreviewCard.test.js [hunk 1]
```diff
@@ -38,7 +39,7 @@ describe('Test suite for ActionButton in BulkConceptsPage component', () => {
             to_concept_code: '',
           },
         ],
-        descriptions: [{ description: '' }],
```
Comment (dkayiwa) [inline src/tests/bulkConcepts/components/PreviewCard.test.js]: Is this change related to the ticket?

File: src/components/bulkConcepts/component/PreviewCard.jsx [hunk 2]
```diff
@@ -61,6 +62,7 @@ const PreviewCard = ({
             </small>
           </p>
           <div className="pop-up-description rounded">{display_name}</div>
+          <CardBody title="Synonym" body={names ? names[1].name : 'N/A'} />
```
Comment (dkayiwa) [inline src/components/bulkConcepts/component/PreviewCard.jsx]: How many names would you expect?

File: src/components/bulkConcepts/component/PreviewCard.jsx [hunk 3]
```diff
@@ -61,6 +62,7 @@ const PreviewCard = ({
             </small>
           </p>
           <div className="pop-up-description rounded">{display_name}</div>
```
Comment (dkayiwa) [inline src/components/bulkConcepts/component/PreviewCard.jsx]: Synonym or Synonyms?

File: src/tests/bulkConcepts/components/PreviewCard.test.js (hunk 1 above)
Comment (AGMETEOR) [reply src/tests/bulkConcepts/components/PreviewCard.test.js]: @dkayiwa this was not supposed to be part of the changes. I have sorted it.

File: src/components/bulkConcepts/component/PreviewCard.jsx (hunk 2 above)
Comment (AGMETEOR) [reply src/components/bulkConcepts/component/PreviewCard.jsx]: @dkayiwa the data that's returned includes key `names` that has an array of objects. The object at index 1 contains the other name for the concept or the synonym.

File: src/components/bulkConcepts/component/PreviewCard.jsx (hunk 3 above)
Comment (AGMETEOR) [reply src/components/bulkConcepts/component/PreviewCard.jsx]: @dkayiwa I now get it. I thought I only had to display one synonym. Let me display all the synonyms of the concept.

File: src/components/bulkConcepts/component/PreviewCard.jsx (hunk 2 above)
Comment (dkayiwa) [reply src/components/bulkConcepts/component/PreviewCard.jsx]: What happens in cases of more than one synonym?

Comment (AGMETEOR) [reply src/components/bulkConcepts/component/PreviewCard.jsx]: @dkayiwa I have changed the implementation to take that into consideration.

Comment (dkayiwa) [reply src/components/bulkConcepts/component/PreviewCard.jsx]: Does the screenshot reflect it?

Comment (AGMETEOR) [reply src/components/bulkConcepts/component/PreviewCard.jsx]: @dkayiwa let me change the screenshot. It had skipped my mind.

Comment (AGMETEOR) [reply src/components/bulkConcepts/component/PreviewCard.jsx]: @dkayiwa I have added the screenshot
